config/config.go

```diff
@@ -227,6 +228,13 @@ func (c *Config) LoadConfigFromContent(content []byte) (err error) {
 	return
 }
 
+// ReadEnvVariables will set qingstor envrionment variables.
+func (c *Config) ReadEnvVariables() (err error) {
```

Comment:
- If this functions is used only in this package, we should not export it.
- `ReadEnvVariables` is not a good name for it's content, I prefer `readCredentialFromEnv`

Reply:
OK